cpp/mysql_diff_schemas.cc

```diff
@@ -72,16 +115,27 @@ void LoadSchema(const std::string &filename,
                                                             ? line.substr(__builtin_strlen("CREATE TABLE "))
                                                             : line.substr(__builtin_strlen("CREATE VIEW ")));
             current_schema.clear();
+        } else if (line_starts_with_create_trigger) {
+            if (not current_schema.empty()) {
+                std::sort(current_schema.begin(), current_schema.end(), SchemaLineIsLessThan);
+                (*table_or_view_name_to_schema_map)[current_table_or_view] = current_schema;
+                current_table_or_view.clear();
+                current_schema.clear();
+            }
+            triggers->emplace_back(line);
         } else {
             if (line[line.length() - 1] == ',')
                 line = line.substr(0, line.length() - 1);
             current_schema.emplace_back(line);
         }
     }
+
     if (not current_schema.empty()) {
         std::sort(current_schema.begin(), current_schema.end(), SchemaLineIsLessThan);
         (*table_or_view_name_to_schema_map)[current_table_or_view] = current_schema;
     }
+
+    std::sort(triggers->begin(), triggers->end(), TriggerLineIsLessThan);
```

Comment:
the std::string that contains a trigger might contain multiple lines, so maybe we should rename this to `TriggerStatementIsLessThan`

Reply:
No because we generate it ourselves with mysql_list_tables.